shogun-lib/src/main/java/de/terrestris/shoguncore/graphql/scalar/GeometryScalar.java

```diff
@@ -0,0 +1,84 @@
+package de.terrestris.shoguncore.graphql.scalar;
+
+import com.bedatadriven.jackson.datatype.jts.JtsModule;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import graphql.language.StringValue;
+import graphql.schema.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.locationtech.jts.geom.Geometry;
+
+import java.util.HashMap;
+
+public class GeometryScalar {
+
+    protected static ObjectMapper objectMapper = (new ObjectMapper()).registerModule(new JtsModule());
+
+    protected final static Logger LOG = LogManager.getLogger(GeometryScalar.class);
+
+    public static final GraphQLScalarType GEOMETRY = new GraphQLScalarType("Geometry", "A custom scalar that handles geometries", new Coercing() {
+
+        @Override
+        public Object serialize(Object dataFetcherResult) {
+            return serializeGeometry(dataFetcherResult);
+        }
+
+        @Override
+        public Object parseValue(Object input) {
+            return parseGeoemtryFromVariable(input);
+        }
+
+        @Override
+        public Object parseLiteral(Object input) {
+            return parseGeometryFromAstLiteral(input);
+        }
+    });
+
+    private static boolean isAGeometry(Object possibleGeometryValue) {
+        return possibleGeometryValue instanceof Geometry;
+    }
+
+    private static Object serializeGeometry(Object dataFetcherResult) {
+        if (isAGeometry(dataFetcherResult)) {
+            Geometry geometry = (Geometry) dataFetcherResult;
+            try {
+                return objectMapper.readValue(objectMapper.writeValueAsString(geometry), HashMap.class);
+            } catch (JsonProcessingException e) {
+                LOG.error("JSON Processing error while writing geometry for GraphQL");
```

Comment:
Maybe you could throw `e` as a `CoercingSerializeException` here as well as in the other overrides?